lists/lists.css

```diff
@@ -0,0 +1,83 @@
+@import url(/CSS/nav.css);
+
+
+.main-container {
+
+    display: flex;
+    flex-direction: column;
+    max-width: 83%;
+    margin-top: 2rem;
+    position: relative;
+    padding-left: 19rem;
+}
+
+h3 {
+
+    margin-bottom: 0.5rem;
+    font-size: 1.8rem;
+
+}
+
+.list-heading {
+    margin-top: 1.3rem;
+    font-weight: bold;
+    font-size: 1.3rem;
+    margin-bottom: 0.5rem;
+}
+
+hr {
+
+    width: 100%;
+    margin: 0.5rem 0rem;
+}
```

Comment:
If it's this general, i.e. an `hr` tag, then firstly maybe avoid using the element/tag selector itself & rather go for a class name as this style should be optional. And secondly, if you want to apply a style to something generic, have it in the main or the utilities CSS file.

Reply:
Thank you, Yash. I'll replace it with a utility class. I missed creating a main.css at the start, hence I plan to do so after finishing the last P1 component